chart/templates/post-delete-hook-cluster-role.yaml

```diff
@@ -12,20 +12,31 @@ rules:
   - apiGroups: [ "extensions","apps" ]
     resources: [ "deployments" ]
     verbs: [ "get", "list", "delete" ]
+  - apiGroups: [ "batch" ]
+    resources: [ "jobs" ]
+    verbs: [ "get", "list", "watch", "delete", "create" ]
   - apiGroups: [ "rbac.authorization.k8s.io" ]
     resources: [ "clusterroles", "clusterrolebindings", "roles", "rolebindings" ]
-    verbs: [ "get", "list", "delete" ]
+    verbs: [ "get", "list", "delete", "create" ]
   - apiGroups: [ "" ]
-    resources: [ "serviceaccounts", "pods", "secrets", "services", "configmaps" ]
+    resources: [ "pods", "secrets", "services", "configmaps" ]
     verbs: [ "get", "list", "delete" ]
+  - apiGroups: [ "" ]
+    resources: [ "serviceaccounts" ]
+    verbs: [ "get", "list", "delete", "create" ]
   - apiGroups: [ "networking.k8s.io" ]
     resources: [ "networkpolicies" ]
     verbs: [ "get", "list", "delete" ]
   - apiGroups: [ "admissionregistration.k8s.io" ]
-    resources: [ "validatingwebhookconfigurations" ]
+    resources: [ "validatingwebhookconfigurations", "mutatingwebhookconfigurations" ]
     verbs: [ "get", "list", "delete" ]
-  - apiGroups: ["policy"]
-    resources: ["podsecuritypolicies"]
-    verbs:     ["use"]
-    resourceNames: [{{ include "rancher.fullname" . }}-post-delete ]
+  - apiGroups: [ "policy" ]
+    resources: [ "podsecuritypolicies" ]
```

Comment:
Actually, do we need to do anything to ensure this whole thing works in hardened cluster or is it already covered ?

Reply:
No, rancher-webhook's newly-added pre-delete-hook does not support harden cluster yet. 
I can add the support in a following PR to rancher-webhook.